Resources/Prototypes/Maps/pebble.yml

```diff
@@ -0,0 +1,46 @@
+﻿- type: gameMap
+  id: pebble
+  mapName: 'Pebble Station'
+  mapPath: /Maps/pebble.yml
+  minPlayers: 0
+  maxPlayers: 30
+  votable: true
+  stations:
+    PebbleStation:
+      mapNameTemplate: '{0} Pebble Station {1}'
+      nameGenerator:
+        !type:NanotrasenNameGenerator
+        prefixCreator: '14'
+      overflowJobs:
+        - Passenger
+      availableJobs:
+        Quartermaster: [ 1, 2 ]
+        SalvageTechnician: [ 2, 2 ]
+        Passenger: [ -1, -1 ]
+        Bartender: [ 1, 1 ]
+        Botanist: [ 1, 2 ]
+        Chef: [ 1, 1 ]
+        Clown: [ 1, 1 ]
+        Janitor: [ 1, 1 ]
+        Mime: [ 1, 1 ]
+        Captain: [ 1, 1 ]
+        HeadOfPersonnel: [ 1, 1 ]
+        ChiefEngineer: [ 1, 1 ]
+        StationEngineer: [ 2, 3 ]
+        ChiefMedicalOfficer: [ 1, 1 ]
+        MedicalDoctor: [ 4, 6 ]
+        Mystagogue: [ 1, 1 ]
+        Epistemologist: [ 1, 3 ]
+        HeadOfSecurity: [ 1, 1 ]
+        SecurityOfficer: [ 2, 2 ]
+        Chaplain: [ 1, 1 ]
+        Warden: [ 1, 1 ]
+        Lawyer: [ 1, 1 ]
+        Musician: [ 1, 1 ]
+        AtmosphericTechnician: [ 1, 1 ]
```

Comment:
Is atlas complex enough here to support this? Otherwise just move this to a normal engineer

Reply:
Leaving atmos tech sorry